torch/_dynamo/variables/builder.py

```diff
@@ -421,6 +422,10 @@ def index_source(key):
                 value,
                 guards=make_guards(GuardBuilder.FUNCTION_MATCH),
             )
+        elif value == is_tracing:
```

Comment:
You should be able to get this behavior with:
https://github.com/pytorch/pytorch/blob/f28927e9c4694819f95c5dda2b111b0ff6dbd606/torch/_dynamo/allowed_functions.py#L192

Reply:
Wait what about skipfiles though? The reason I did this was to override skipfiles